www/nginx/src/opnsense/service/templates/OPNsense/Nginx/location.conf

```diff
@@ -147,6 +147,7 @@ location {{ location.matchtype }} {{ location.urlpattern }} {
 {% endif %}
     proxy_ignore_client_abort {% if location.proxy_ignore_client_abort == '1' %}on{% else %}off{% endif %};
     proxy_request_buffering {% if location.proxy_request_buffering == '1' %}on{% else %}off{% endif %};
+    proxy_max_temp_file_size {% if location.proxy_max_temp_file_size is defined %}{{ location.proxy_max_temp_file_size }}{% else %}1024{% endif %}m;
```

Comment:
I recommend checking for the condition where location.proxy_max_temp_file_size == "0" and removing the trailing "m" from the command if it is. It should read:
proxy_max_temp_file_size 0;
**NOT:**
proxy_max_temp_file_size 0m;

Reply:
0 with any suffix is still 0. It should not make a difference.

Reply:
Perhaps I'm just being OCD 😁